torch/_inductor/graph.py

```diff
@@ -277,7 +278,12 @@ def constant_name(self, name: str, device_override: torch.device):
         return alt_name
 
     def placeholder(self, target: str, args, kwargs):
-        example: torch.Tensor = super().placeholder(target, args, kwargs)
+        example = super().placeholder(target, args, kwargs)
+        if not torch.is_tensor(example):
+            example = GenericBox(example)
+            self.graph_inputs[target] = example
+            self.graph_inputs_original[target] = example.data
+            return example
```

Comment:
`SymInt` value now appears here, so we wrap it into `GenericBox` to be able to distinguish tensors and non-tensors in other functions.